build.xml

```diff
@@ -2,7 +2,7 @@
 
 <project name="Profile-ci" default="build">
     <target name="build"
-            depends="php-lint-ci,phpmd-ci,phpcs-ci,phpcpd-ci,phpunit-ci,php-security-checker"/>
+            depends="php-lint-ci,phpmd-ci,phpcs-ci,phpcpd-ci,phpunit-ci,php-security-checker,npm-audit"/>
```

Comment:
May I suggest merging `php-security-checker` and `npm-audit` in a single `security-check` task. As we do in most other projects? This works fine too, but running them together is even better in my opinion.

Reply:
For this single task I couldn't find any added befit to run them both at once. Because you normally want to run the checkers for npm or composer separately or you want to run all the tests together. Would you please elaborate why you want to run all security checks together so I can make a better decision in the future? Thanks!

Reply:
You can keep them as separate tasks, but providing a task that combines them makes it more convenient to run them in your dev env. Basically the same reason why we bundle the different static analysis tasks.